source/templates.tex

```diff
@@ -1945,7 +1942,7 @@
 have the same type and value
 after conversion to the type of the \grammarterm{template-parameter},
 where they are considered to have the same value if they compare equal
-with \tcode{operator<=>}, and}
+with \tcode{operator==}, and}
```

Comment:
Pre-existing: `\tcode{operator==}` seems to only apply to class types. Shouldn't this say "if they compare equal with the \tcode{==} operator"?

Reply:
Sounds reasonable. I'll also put in a cross-reference to [expr.eq].